apps/src/templates/CodeWorkspace.jsx

```diff
@@ -57,7 +58,8 @@ class CodeWorkspace extends React.Component {
         if (
```

Comment:
I can't comment on the line above, but should we add a comment here about what `hasIncompatibleSources` is to match the others?

Reply:
yep will do!